src/makeCalculator.js

```diff
@@ -4,7 +4,55 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+
+    operate(callback, operand) {
+      this.result = callback(this.result, operand);
```

Comment:
<img width="821" alt="Screenshot 2024-07-09 at 15 32 41" src="https://github.com/mate-academy/js_advanced_calculator/assets/135232001/69218730-cac9-4aa4-b32a-023240ed53b3">
Feel free to ask for help in the chat